### ISSUE TITLE
BUG - WEBFORMS - Select options are not displayed in subform when another subform exists into it

### ISSUE THREAD

**Issue by atl2017** [smap-consulting/smapserver#596]
### Actual Behaviour: In the following form, row 9 Activity (select_one) does not display options if there is more than one subforms (actions) when begin cycle 2:

![image](https://user-images.githubusercontent.com/29519901/79625753-c131e380-80f0-11ea-8467-1301b7ab575f.png)

Example: In the following case there is just one action (one subform) into activity 1 (external repeat), so second select options are displayed
![image](https://user-images.githubusercontent.com/29519901/79625812-30a7d300-80f1-11ea-91e3-015d01271cf7.png)
But if there are two actions, next select_one  options are not displayed:

![image](https://user-images.githubusercontent.com/29519901/79625867-a90e9400-80f1-11ea-9330-4967fd17cc8c.png)
If there is jus one action in the first  external cycle (activities), one action in the second cycle, then third cycle fails:
![image](https://user-images.githubusercontent.com/29519901/79625943-2508dc00-80f2-11ea-8d31-a52ab12969f0.png)

### Steps to reproduce the problem: 
Test 1:
External repeat: select one activity - Cycle 1
    Internal repeat: write one action
External repeat: select one activity - Cycle 2
    Internal repeat: write one action
External repeat: select one activity - Cycle 3
   
It should be fail to display choices:

Test 2
External repeat: select one activity - Cycle 1
    Internal repeat: write one action
    Internal repeat: write second action
External repeat: select one activity - Cycle 2 -> It should be failed to display choices

**Comment by Rajen2020** [comment]
Yes, I am also having similar issue with regards looking up data from subform when there is a nested subform. Please consider looking at this.
